OMNotebook/OMNotebook/OMNotebookGUI/cellcommands.cpp

```diff
@@ -519,6 +514,7 @@ namespace IAEX
          (document()->getCursor())->deleteCurrentCell();
       }
    }
+   application()->clearPasteboard();
```

Comment:
Can we move this above the for loop block and after the line `document()->clearSelection();` to keep it consistent with rest of the code.